test/systemtest/conftest.py

```diff
@@ -27,6 +28,25 @@ def pytest_exception_interact(node, call, report):
 
     utils.dump_temp_files(node.funcargs['tmp_path_factory'].getbasetemp())
 
+
+@pytest.fixture(scope="session")
+def localconf(request):
+    """Host-local configuration."""
+    if os.getenv('OPENTITAN_TEST_LOCALCONF') and os.path.isfile(
+            os.environ['OPENTITAN_TEST_LOCALCONF']):
+        localconf_yaml_file = os.environ['OPENTITAN_TEST_LOCALCONF']
+    else:
+        XDG_CONFIG_HOME = os.getenv(
+            'XDG_CONFIG_HOME', os.path.join(os.environ['HOME'], '.config'))
+        localconf_yaml_file = os.path.join(XDG_CONFIG_HOME, 'opentitan',
+                                           'test-localconf.yaml')
```

Comment:
I think this would probably be better if we looked at environment variables to figure out the path (in the same order), and picked the first path that had the relevant environment variables set. Then we can shout if the config file doesn't exist. I think that otherwise, I'll be very confused if I have (say) one configuration in `~/.config/opentitan/opentitan/test-localconf.yaml` and then set `OPENTITAN_TEST_LOCALCONF` with a typo. "Why isn't it getting my changes?!!" etc.

Reply:
Just checking: this has gone away completely now, right?

Reply:
No it hasn't gone away and good that you ask, somehow I missed that comment (GH really doesn't make it easy to miss things ...). Any yes, `isfile()` should go, will be in the next update.

Reply:
Oh, how weird: I couldn't find the code any more! Cool, thanks :-)